### PR TITLE
DBZ-7820 Adding documentation for Postgres custom data type converters

### PR DESCRIPTION
**Pull request by zeldanerd24**
https://issues.redhat.com/browse/DBZ-7820

### CODE REVIEW

**Review by Naros — COMMENTED** [review]
This LGTM ,can you review @roldanbob as time permits.

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 1]
```diff
@@ -1938,6 +1939,17 @@ Support for the propagation of default values exists primarily to allow for safe
...
 ====
 
+[[custom-postgres-connector-converters]]
+== Custom converters
+
+If your database contains custom data types, such as composite types created with `CREATE TYPE` statements, Debezium will not replicate data from columns with those types by default.
+To replicate those columns, the basic instructions for {link-prefix}:{link-custom-converters}#custom-converters[creating a custom converter] apply, with a few important caveats:
+
+* You must set xref:postgresql-property-include-unknown-datatypes[`include.unknown.datatypes`] to `true`, otherwise values passed to the custom converter will always be `null`
+* The type of value passed to the converter will vary based on the logical decoding output plug-in
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:1949]:
```suggestion
* The type of value that is passed to the converter depends on the logical decoding output plug-in that is configured for the replication slot.
```

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 2]
```diff
@@ -1938,6 +1939,17 @@ Support for the propagation of default values exists primarily to allow for safe
...
+** `decoderbufs` will get a byte array (`byte[]`) representation of the column data
+** `pgoutput` will get a string representation of the column data
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:1951]:
```suggestion
** `decoderbufs` passes a byte array (`byte[]`) representation of the column data.
** `pgoutput` passes a string representation of the column data.
```

**Review by roldanbob — CHANGES_REQUESTED** [review]
Thanks, @zeldanerd24! A few items for your consideration. Let me know if you have any questions.

**Comment by jpechane** [comment]
@zeldanerd24 Applied, thanks

**Pull request merged**